### DISCUSSION TITLE
ObservableProperty marked fields and Property Attributes

### DISCUSSION THREAD

**Discussion by luis-cortina** [CommunityToolkit/dotnet#522] [Q&A] [answered]
I have the need to be able to add Attributes to properties generated by fields marked with the ObservableProperty.
Is this in any way supported? i.e. I add the attribute to the field and it is generated in the property?

Thanks in advance.

**Comment by pictos** [comment]
@Sergio0694 I believe this discussion is around the MVVM source generators, if so could you move the right repo?
(I don't have the power to do so)

**Reply by Sergio0694** [reply]
Thank you for the ping! 🙂

**Reply by pictos** [reply]
Thanks for the quick reply

**Comment by Sergio0694** [comment, marked as answer]
@luis-cortina This is now supported in 8.0.0-preview1, see https://github.com/CommunityToolkit/dotnet/issues/413 😄
